paddle/contrib/inference/demo_ci/run.sh

```diff
@@ -2,7 +2,12 @@ set -x
 PADDLE_ROOT=$1
 WITH_MKL=$2
 WITH_GPU=$3
-if [ $3 == "ON" ]; then
+if [ $2 == ON ]; then
+  # You can export yourself if move the install path
+  MKL_LIB=${PADDLE_ROOT}/build/fluid_install_dir/third_party/install/mklml/lib
+  export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${MKL_LIB}
+fi
+if [ $3 == ON ]; then
```

Comment:
This fix 
https://paddleci.ngrok.io/viewLog.html?buildId=1692&tab=buildLog&buildTypeId=Paddle_PrCi&logTab=tree&filter=all&_focus=15022
```
./simple_on_word2vec: error while loading shared libraries: libmklml_intel.so: cannot open shared object file: No such file or directory
```